ajax/libs/markerclustererplus/2.0.15/markerclusterer.min.js

```diff
@@ -0,0 +1,2 @@
+//# sourceMappingURL=markerclusterer.min.js.map
```

Comment:
@hare1039 Run `minify.sh` with augment `--no-map`. You can try to read the code in `minify.sh`.